### PR TITLE
Bloom's example for fast linear regression

### PR DESCRIPTION
**Pull request by mortendahl**

### CODE REVIEW

**Comment by mortendahl** [comment]
Reference R code updated: https://github.com/jbloom22/DASH/blob/master/dash.r

File: examples/bloom/regressor.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    for attr, partial_estimator in zip(self.components,
+                                       zip(*partial_estimators)):
+      setattr(self, attr, tfe.add_n(partial_estimator))
+
+    with tfe.Session() as sess:
+      for k in self.components:
+        op = getattr(self, k)
+        setattr(self, k, sess.run(op.reveal()))
+
+    self._inverted_covariate_square = tf.linalg.inv(self.covariate_square)
```
Comment (yanndupis) [inline examples/bloom/regressor.py]: Creating a new graph should fix your device error:

```
tf_graph = tf.Graph()
with tf_graph.as_default():
      self._inverted_covariate_square = tf.linalg.inv(self.covariate_square)
      self.coefficients = tf.matmul(self._inverted_covariate_square,
                                    self.covariate_label_product)
```

File: examples/bloom/regressor.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    self.coefficients = tf.matmul(self._inverted_covariate_square,
+                                  self.covariate_label_product)
+    with tf.Session() as sess:
```
Comment (yanndupis) [inline examples/bloom/regressor.py]: You can call the new graph with `tf.Session(graph=tf_graph)`

**Comment by mortendahl** [comment]
Let me know if you want me to finish this one up @jvmancuso, seems like it's only minor things left!

File: examples/bloom/regressor.py (hunk 1 above)
Comment (jvmancuso) [reply examples/bloom/regressor.py]: oof

**Comment by jvmancuso** [comment]
should be good for review @mortendahl

**Comment by mortendahl** [comment]
Let's merge this guy up, great work so far! I'll create a few issues that we can pick up later.

**Pull request merged**